picocli/src/main/java/io/micronaut/configuration/picocli/PicocliRunner.java

```diff
@@ -43,6 +52,9 @@
     accessType = {TypeHint.AccessType.ALL_DECLARED_FIELDS, TypeHint.AccessType.ALL_PUBLIC_METHODS, TypeHint.AccessType.ALL_DECLARED_CONSTRUCTORS}
 )
 public class PicocliRunner {
+
+    private PicocliRunner() {}
```

Comment:
adding a private constructor, as Utility class should not have public constructor

Reply:
this might be a breaking change, not sure if it will be allowed by the binary compatibility checks

Reply:
OK, then I'll rollback it